README.md

```diff
@@ -57,6 +57,8 @@ vars:
     using_company_tags: False
     using_contact_tags: False
     using_conversation_tags: False
+    using_team: False
```

Comment:
The team admin table is only necessary to join the admins to the respective team. I think it would be best to consolidate the two metrics into one `using_teams` variable that is used across team and team admin. This will also allow for a single conditional statement in the `intercom__admin_metrics` model.